lib/internal/modules/cjs/loader.js

```diff
@@ -1021,6 +1039,8 @@ Module._load = function(request, parent, isMain) {
     }
```

Comment:
We might also want to end the timer in the throw statement above? Not sure what it would look like in the trace if someone try-catch it...

Reply:
Good catch